### DISCUSSION TITLE
Invalid plots / Chia 1.1.6 CLI / "Failed to open file ... ValueError: Invalid number of bytes 2"

### DISCUSSION THREAD

**Discussion by kettlewarrior** [Chia-Network/chia-blockchain#6210] [Support]
Hi,

I am using Chia CLI 1.1.6  on Ubuntu 20.4. I do not use a GUI.

Any plot I create seems to be rejected with the error:

```
$ chia plots check
```

```
2021-05-29T20:20:16.223  chia.plotting.plot_tools         : ERROR    Failed to open file /mnt/vol4/chia/plot<snip>.plot. Invalid number of bytes 2 Traceback (most recent call last):
  File "/home/<snip>/chia-blockchain/chia/plotting/plot_tools.py", line 208, in process_file
    ) = parse_plot_info(prover.get_memo())
  File "/home/<snip>/chia-blockchain/chia/plotting/plot_tools.py", line 82, in parse_plot_info
    raise ValueError(f"Invalid number of bytes {len(memo)}")
ValueError: Invalid number of bytes 2
```

I log each plot run, with the keys for the invalid plot always matching those in the output from ```chia keys show```.

I feel like I have missed something obvious somewhere (some step, some dependancy), but cannot find what despite searching.

Deleting Chia and starting again form scratch did not help, either:

```
chia stop all
sudo pip remove chia-blockchain chiabip158 chiapos chiavdf
rm -rf ~/.chia
rm -rf chia-blockchain
```

Any help appreciated! Thanks!

**Comment by tiny0little** [comment]
could you post full command `chia plots create` options you use?

**Reply by kettlewarrior** [reply]
Thanks for taking a look! :-)

Sample command:

```nohup chia plots create -n 1 -m 8192 -r 2 -t /mnt/vol1/scratch -d /mnt/vol4/chia -x > ~/chia/logs/bench_grp1_r2_m8_vol1.log 2>&1 &```

- have also used different permutations of memory, threads, scratch space etc.
- have also run plotting processes alone / concurrently with different volumes as the scratch target

**Reply by tiny0little** [reply]
there's no definition of farmer/pool key. do you have the key installed on this host? more than one?
`chia keys show` shows it/them? any of the plots recognized or all give the error?

I use `chia plots create -a <fingerprint>` , the fingerprint from `chia keys show`

**Reply by kettlewarrior** [reply]
Yes – the keys are on the host and are present in the resulting log.

From ```chia keys show```

```
Master public key (m): a3b3b40bb3b79c1c41b28be5b12a2ea0f4a0f21efbca8a6a5ce0eacef7255ce5d128defe50654e9f3392010cb0aa1389
Farmer public key (m/12381/8444/0/0): b8e0dd8fbac06b33ab78e24f44a6113d551b389c43151e2eddad69111a47946552f3e51446b3a8dcebc7001477c7b027
Pool public key (m/12381/8444/1/0): aa3b8b4b4b09dca9444cc8091004f2050af8a4bead42486a5e794d6af45689dadf79d143fd61665417f6533a1d8f4909
```

From the ```chia plots create``` log file:

```
2021-05-29T13:44:01.489  chia.plotting.create_plots       : INFO     Creating 1 plots of size 32, pool public key:  aa3b8b4b4b09dca9444cc8091004f2050af8a4bead42486a5e794d6af45689dadf79d143fd61665417f6533a1d8f4909 farmer public key: b8e0dd8fbac06b33ab78e24f44a6113d551b389c43151e2eddad69111a47946552f3e51446b3a8dcebc7001477c7b027
2021-05-29T13:44:01.496  chia.plotting.create_plots       : INFO     Debug memo: 8192
2021-05-29T13:44:01.497  chia.plotting.create_plots       : INFO     Memo: 8192
2021-05-29T13:44:01.497  chia.plotting.create_plots       : INFO     NOT adding directory /mnt/vol4/chia to harvester for farming
2021-05-29T13:44:01.499  chia.plotting.create_plots       : INFO     Starting plot 1/1
```

In the meantime – I created a new Linux user and started from scratch. The plots generated from the new account with the newly generated keys etc. work.

Very curious about what I have corrupted / confused to generate this behaviour (... and how!).

Incidentally, the host machine was flattened before I did anything with Chia (the host will only run Chia). Should be nothing abnormal in the general configuration as a consequence.

```
$ cat /proc/version
Linux version 5.4.0-73-generic (buildd@lcy01-amd64-019) (gcc version 9.3.0 (Ubuntu 9.3.0-17ubuntu1~20.04)) #82-Ubuntu SMP Wed Apr 14 17:39:42 UTC 2021
```

**Comment by BrandtH22** [comment]
@kettlewarrior , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
